GettingToKnowPython/HomeWorks/HomeWork_1/Task1(2).py

```diff
@@ -0,0 +1,15 @@
+"""Задача 2: Найдите сумму цифр трехзначного числа.
+*Пример:*
+123 -> 6 (1 + 2 + 3)
+100 -> 1 (1 + 0 + 0) |
+"""
+N = 257 #n - трехзначное число
+n = int(N)
```

Comment:
Погодите, так переменная `n` уже в целочисленном типе. Python – язык сценической типизацией. Какой тип данных вы присвоите переменной, такой тип данных она и будет содержать до следующего присвоения.
```suggestion
n = 257 #n - трехзначное число
```